invenio_app_ils/circulation/api.py

```diff
@@ -44,7 +44,7 @@
 from invenio_app_ils.minters import pid_minter
 from invenio_app_ils.proxies import current_app_ils
 
-lt_es7 = ES_VERSION[0] < 7
+lt_es7 = not uses_es7
```

Comment:
It would be better to change the occurrences of `lt_es7` and remove completely `lt_es7`, instead of introducing a negation of another variable.